src/modules/SX126x.cpp

```diff
@@ -721,6 +721,33 @@ int16_t SX126x::setSyncWord(uint8_t* syncWord, uint8_t len) {
   return(state);
 }
 
+int16_t SX126x::setSyncBits(uint8_t *syncWord, uint8_t bitsLen) { 
+  // check active modem
+  if(getPacketType() != SX126X_PACKET_TYPE_GFSK) {
+    return(ERR_WRONG_MODEM);
+  }
+
+  // check sync word Length
+  if(bitsLen > 0x40) {
+    return(ERR_INVALID_SYNC_WORD);
+  }
+
+  uint8_t bytesLen = bitsLen / 8;
+  if ((bitsLen % 8) != 0) bytesLen++;
```

Comment:
Could you add the braces. as explained in [code style guideline](https://github.com/jgromes/RadioLib/blob/master/CONTRIBUTING.md#code-style-guidelines)?

Reply:
Sorry, I forgot about this one. Will repair ASAP.